build.gradle.kts

```diff
@@ -1,167 +1,119 @@
+import dev.kordex.gradle.plugins.kordex.DataCollection
 import java.util.*
 
 plugins {
-    application
-
-    kotlin("jvm")
-    kotlin("plugin.serialization")
-
-    id("com.github.gmazzo.buildconfig") version "3.1.0"
-    id("com.github.jakemarsden.git-hooks")
-    id("com.github.johnrengelman.shadow")
-    id("io.gitlab.arturbosch.detekt")
-    id("net.kyori.blossom") version "2.1.0"
-    id("net.kyori.indra.git") version "3.1.3"
+	application
+
+	kotlin("jvm")
+	kotlin("plugin.serialization")
+
+	id("com.github.gmazzo.buildconfig")
+	id("com.github.jakemarsden.git-hooks")
+	id("com.github.johnrengelman.shadow")
+	id("dev.kordex.gradle.kordex")
+	id("io.gitlab.arturbosch.detekt")
+	id("net.kyori.blossom")
+	id("net.kyori.indra.git")
 }
 
 fun String.runCommand(
-    workingDir: File = File("."),
-    timeoutAmount: Long = 60,
-    timeoutUnit: TimeUnit = TimeUnit.SECONDS
+	workingDir: File = File("."),
+	timeoutAmount: Long = 60,
+	timeoutUnit: TimeUnit = TimeUnit.SECONDS
 ): String = ProcessBuilder(split("\\s(?=(?:[^'\"`]*(['\"`])[^'\"`]*\\1)*[^'\"`]*$)".toRegex()))
-    .directory(workingDir)
-    .redirectOutput(ProcessBuilder.Redirect.PIPE)
-    .redirectError(ProcessBuilder.Redirect.PIPE)
-    .start()
-    .apply { waitFor(timeoutAmount, timeoutUnit) }
-    .run {
-        val error = errorStream.bufferedReader().readText().trim()
-        if (error.isNotEmpty()) {
-            throw Exception(error)
-        }
-        inputStream.bufferedReader().readText().trim()
-    }
+	.directory(workingDir)
+	.redirectOutput(ProcessBuilder.Redirect.PIPE)
+	.redirectError(ProcessBuilder.Redirect.PIPE)
+	.start()
+	.apply { waitFor(timeoutAmount, timeoutUnit) }
+	.run {
+		val error = errorStream.bufferedReader().readText().trim()
+		if (error.isNotEmpty()) {
+			throw Exception(error)
+		}
+		inputStream.bufferedReader().readText().trim()
+	}
 
 
 group = "org.hyacinthbots.allium"
-version = "0.7-build.local-" + "git rev-parse --short=8 HEAD".runCommand(workingDir = rootDir) + "-" + "git branch --show-current".runCommand(workingDir = rootDir).replace("/", ".")
+
+version = "0.8-build.local-" +
+	"git rev-parse --short=8 HEAD"
+		.runCommand(workingDir = rootDir) +
+	"-" +
+	"git branch --show-current"
+		.runCommand(workingDir = rootDir)
+		.replace("/", ".")
+
 var buildTime = Date().time / 1000
-// version = "0.6.5"
```

Comment:
this version was commented so the release can have a clean version instead of the one with git rev-parse etc as visible in the current about command